packages/admin-next/dashboard/src/components/layout/notifications/notifications.tsx

```diff
@@ -31,10 +57,93 @@ export const Notifications = () => {
       </Drawer.Trigger>
       <Drawer.Content>
         <Drawer.Header>
-          <Heading>Notifications</Heading>
+          <Heading>{t("notifications.domain")}</Heading>
         </Drawer.Header>
-        <Drawer.Body>Notifications will go here</Drawer.Body>
+        <Drawer.Body className="overflow-y-auto">
```

Comment:
You can remove the padding here, by setting `p-0` or `py-0 px-0` (can't remember what class is used to set the padding on the component in UI, and you need to override the specific one(s)).